src/components/InputProblem.tsx

```diff
@@ -28,16 +30,24 @@ export const InputProblem = ({
   const schema = (): Yup.SchemaOf<InputSchema> => {
     if (comparator.toLowerCase() === 'integer') {
       return Yup.object({
-        response: Yup.number().integer('Please provide an Integer').typeError('Please provide an Integer').required('Please provide an Integer')
+        response: Yup.number()
+          .integer('Please provide an Integer')
+          .typeError('Please provide an Integer')
+          .required('Please provide an Integer')
```

Comment:
Both to keep this PR clean and to minimize potential merge conflicts with the PR @P-Gill97 is working on, I'd revert this altogether so no diff appears.